src/nexmo/verify.py

```diff
@@ -0,0 +1,89 @@
+import nexmo
+import warnings
+
+class Verify:
+    def __init__(
+        self,
+        client=None,
+        key=None,
+        secret=None
+    ):
+        try:
+            self._client = client
+            if self._client is None:
+                self._client = nexmo.Client(
+                    key=key,
+                    secret=secret
+                )
+        except Exception as e:
+            print('Error: {error_message}'.format(error_message=str(e)))
+    
+    def request(self, params=None, **kwargs):
+        return self._client.post(self._client.api_host(), "/verify/json", params or kwargs)
+    
+    def send_verification_request(self, params=None, **kwargs):
```

Comment:
as this was deprecated in the old module can we remove this from the new module?